modules/images-configuration-parameters-hcp.adoc

```diff
@@ -0,0 +1,43 @@
+// Module included in the following assemblies:
+//
+// * openshift_images/image-configuration-hcp.adoc
+
+[id="images-configuration-parameters-hcp_{context}"]
+= Image controller configuration parameters for {hcp-title}
+
+The `image.config.openshift.io/cluster` resource holds cluster-wide information about how to handle images. The resource exists, but it is read only and can only be changed through supported tools like ROSA CLI (`rosa`). The canonical, and only valid name is `cluster`. It can be configured in {product-rosa} {hcp} through ROSA CLI (`rosa`) commands.
```

Comment:
```suggestion
The `image.config.openshift.io/cluster` resource holds cluster-wide information about how to handle images. The resource exists, but it is read only and can only be changed through supported tools like ROSA CLI (`rosa`). The canonical and only valid name is `cluster`. It can be configured in {product-rosa} {hcp} through ROSA CLI (`rosa`) commands.
```